taglib/mpeg/mpegheader.h

```diff
@@ -29,6 +29,8 @@
 #include "taglib.h"
 #include "taglib_export.h"
 
+#include <memory>
+
```

Comment:
I noticed that `memory` is included here but not in most of the other headers. For consistency it should probably be included in all of them that use `shared_ptr`.

Reply:
Right. I did this as it seems most cop files use includes inherited from elsewhere.

Maybe include-what-you-use should be ran against the codebase.

Reply:
That sounds like a great candidate for a PR once I get the current few merged.

Reply:
You still want me to add memory everywhere in this PR?

Reply:
I think it's a good technique but not necessary for this PR. I'm not aware of a taglib style for includes.